docker/linux-trusty-gcc4.8-tapir5.0/Dockerfile

```diff
@@ -19,12 +19,12 @@ ENV CXX /usr/bin/g++
 ENV LLVM_SOURCES /tmp/llvm_sources-tapir5.0
 WORKDIR $LLVM_SOURCES
 
-ENV CLANG_PREFIX /usr/local/clang+llvm-tapir5.0
+ENV CLANG_PREFIX $HOME/clang+llvm-tapir5.0
 ENV CMAKE_VERSION cmake
 
 RUN git clone --recursive https://github.com/wsmoses/Tapir-LLVM llvm && \
     mkdir -p ${LLVM_SOURCES}/llvm_build && cd ${LLVM_SOURCES}/llvm_build && \
-    ${CMAKE_VERSION} -DCMAKE_INSTALL_PREFIX=${CLANG_PREFIX} -DLLVM_TARGETS_TO_BUILD=X86 -DCOMPILER_RT_BUILD_CILKTOOLS=OFF -DLLVM_ENABLE_CXX1Y=ON -DLLVM_ENABLE_TERMINFO=OFF -DLLVM_BUILD_TESTS=OFF -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release -DLLVM_BUILD_LLVM_DYLIB=ON  -DLLVM_ENABLE_RTTI=ON ../llvm/ && \
+    ${CMAKE_VERSION} -DLLVM_REQUIRES_EH=ON -DLLVM_ENABLE_OCAMLDOC=OFF -DLLVM_INSTALL_OCAMLDOC_HTML_DIR=/tmp -DLLVM_OCAML_INSTALL_PATH=/tmp -DCMAKE_INSTALL_PREFIX=${CLANG_PREFIX} -DLLVM_TARGETS_TO_BUILD=X86 -DCOMPILER_RT_BUILD_CILKTOOLS=OFF -DLLVM_ENABLE_CXX1Y=ON -DLLVM_ENABLE_TERMINFO=OFF -DLLVM_BUILD_TESTS=OFF -DLLVM_ENABLE_ASSERTIONS=ON -DCMAKE_BUILD_TYPE=Release -DLLVM_BUILD_LLVM_DYLIB=ON  -DLLVM_ENABLE_RTTI=ON ../llvm/ && \
```

Comment:
I thought the flag was LLVM_ENABLE_EH. Are you sure it's REQUIRES_EH?

Reply:
both are supported
https://github.com/wsmoses/Tapir-LLVM/blob/6d439c5b676577f48e20d54edef477d1a6c517bc/cmake/modules/AddLLVM.cmake#L13

Reply:
but I am not sure why there exists two flags though, and it feels like REQUIRES_EH might be enabling something more?